projects/plugins/jetpack/class.jetpack.php

```diff
@@ -1213,7 +1218,7 @@ public function register_assets() {
 				'jetpack-facebook-embed',
 				Assets::get_file_url_for_environment( '_inc/build/facebook-embed.min.js', '_inc/facebook-embed.js' ),
 				array(),
-				null,
+				'20210107',
```

Comment:
Could we use `JETPACK__VERSION` here? I know we don't update that file too often, but I worry that when we next update the file, we'll forget to update this version at the same time.

```suggestion
				JETPACK__VERSION,
```